src/ops/info.rs

```diff
@@ -50,14 +51,24 @@ pub fn info(
 
     validate_locked_and_frozen_options(package_id, config)?;
 
-    let current_rustc_version = &config.load_global_rustc(ws.as_ref())?.version;
-    // Remove any pre-release identifiers for easier comparison.
-    // Otherwise, the MSRV check will fail if the current Rust version is a nightly or beta version.
-    let current_rustc_version = &semver::Version::new(
-        current_rustc_version.major,
-        current_rustc_version.minor,
-        current_rustc_version.patch,
-    );
+    let msrv_from_nearest_manifest_or_ws =
+        try_get_msrv_from_nearest_manifest_or_ws(config, ws.as_ref());
+    // If the workspace does not have a specific Rust version,
+    // or if the command is not called within the workspace, then fallback to the global Rust version.
+    let rustc_version = match msrv_from_nearest_manifest_or_ws {
+        Some(msrv) => msrv,
+        None => {
+            // Remove any pre-release identifiers for easier comparison.
+            // Otherwise, the MSRV check will fail if the current Rust version is a nightly or beta version.
```

Comment:
nit: shouldn't this comment be on the next line?

Reply:
Updated.

Reply:
Looking at https://github.com/hi-rustin/cargo-information/compare/5092c82548b6eebe0afc5ff8dd491b5dbff5b064..b2e030de9a8242fbcb0efff21d51d1a38e2039bb I think the wrong comment was  moved

Reply:
I apologize, I must be blind.

Reply:
Updated.